docs/reference/api.yml

```diff
@@ -352,6 +352,27 @@ components:
       name: categories_tags_en
       description: |
         The category of products you are searching for.
+    sort_by:
+      schema:
+        type: string
+        example: product_name
+        enum:
+          - product_name
+          - last_modified_t
+          - last_modified_t_complete_first
```

Comment:
this seems deprecated 
@stephanegigandet may confirm, see Display.pm line 4719.

```suggestion
```

Reply:
Yes, let's remove it from the doc.